matrix-sdk-android/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         jcenter()
     }
     dependencies {
```

Comment:
More lib can be updated here 
androidx.recyclerview:recyclerview -> 1.2.0-xx ? (may fix some reported RS?)